plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchTaskHandler.groovy

```diff
@@ -164,6 +171,13 @@ class AzBatchTaskHandler extends TaskHandler {
         return taskState
     }
 
+    /**
+     * @return Retrieve the executionInfo for the executed task
+     */
+    protected TaskExecutionInformation taskExecutionInfo(AzTaskKey key) {
+        return batchService.getTask(key).executionInfo()
+    }
```

Comment:
Bit redundant having a method only for this. It'd ore clear just using `batchService.getTask(key).executionInfo()`

Reply:
removed